package.json

```diff
@@ -86,6 +86,7 @@
     "build:server": "NODE_ENV=production webpack --progress --mode production --config config/webpack.server.config.js",
     "codegen": "graphql-codegen -r dotenv/config dotenv_config_path=.env.development.local  --config codegen.yml",
     "test": "react-scripts test --env=jest-environment-jsdom-sixteen",
+    "test:all": "set CI=true && react-scripts test --env=jest-environment-jsdom-sixteen --watchAll=false && yarn lint",
```

Comment:
maybe we should lint first, so we can fail faster? Tests take pretty long

Reply:
yea, we can